ext/InverseFunctionsDistributionsExt.jl

```diff
@@ -0,0 +1,17 @@
+module InverseFunctionsDistributionsExt
+
+import InverseFunctions: inverse
+using Distributions
+
+
+for (fcdf, fqua) in [
+    (:cdf, :quantile),
+    (:ccdf, :cquantile),
+    (:logcdf, :invlogcdf),
+    (:logccdf, :invlogccdf),
+]
+    @eval inverse(f::Base.Fix1{typeof($fcdf), <:ContinuousUnivariateDistribution}) = Base.Fix1($fqua, f.x)
```

Comment:
In general, that's not correct, `x -> quantile(dist, x)` does not necessarily invert the result of `x -> cdf(dist, x)`:
```julia
julia> using Distributions

julia> quantile(Gamma(), cdf(Gamma(), -10))
0.0
```